### ISSUE TITLE
 Invalid API key

### ISSUE THREAD

**Issue by kk-atom** [amplitude/Amplitude-Kotlin#215]
API key is correct but SDK says that its not. Org ID 100002402. Also fix this network request spam.
```
2024-08-09 17:18:28.690 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:28.897 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:28.897 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:28.897 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:29.116 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:29.116 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:29.116 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:29.324 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:29.325 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:29.325 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:29.534 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:29.535 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:29.535 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:29.751 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:29.752 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:29.752 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:29.974 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:29.975 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:29.975 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:30.181 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:30.182 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:30.182 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:30.417 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:30.417 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:30.417 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:30.629 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:30.629 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:30.629 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:30.929 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:30.930 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:30.930 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:31.136 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:31.136 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:31.136 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:31.352 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:31.353 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:31.353 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:31.748 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:31.748 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:31.748 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:31.980 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:31.981 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:31.981 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:32.368 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:32.369 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:32.369 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:32.618 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:32.618 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:32.618 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:32.977 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:32.977 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:32.977 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:33.194 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:33.195 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:33.195 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:33.565 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:33.566 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:33.566 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:33.777 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:33.778 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:33.778 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:34.000 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:34.001 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:34.001 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:34.209 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:34.210 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
2024-08-09 17:18:34.210 10497-10603 Amplitude               apar.ride.app                        D  Max retries 5 exceeded, temporarily stop scheduling new events sending out.
2024-08-09 17:18:34.415 10497-10603 Amplitude               apar.ride.app                        D  Handle response, status: BAD_REQUEST, error: Invalid API key: 6eaecc157d0ff5a58c0b0d6df54307be
2024-08-09 17:18:34.415 10497-10603 Amplitude               apar.ride.app                        D  Back off to retry sending events later.
```

**Comment by izaaz** [comment]
@kk-atom the spam is fixed in version 1.7.1.

Wrt invalid API key, it looks like the project is an EU project. Have you configured the SDK (`serverZone`) to send data to the EU endpoint?

https://amplitude.com/docs/sdks/analytics/android/android-kotlin-sdk#configure-the-sdk

**Comment by kk-atom** [comment]
Thanks
